srcpkgs/LibreCAD/template

```diff
@@ -46,9 +46,14 @@ do_install() {
 	vcopy librecad.svg usr/share/icons/hicolor/scalable/apps
 
 	# install resources
-	for resource in library patterns fonts qm; do
+	for resource in fonts library patterns qm; do
 		vcopy unix/resources/${resource} usr/share/librecad/
 	done
 
-    vcopy unix/resources/plugins usr/lib/librecad/
+	# install plugins
+	vmkdir usr/share/librecad/plugins
+	vcopy unix/resources/plugins usr/lib/librecad/
+	for plugins in libalign libasciifile libimportshp liblist libpicfile libplotequation libsameprop libsample; do
+		ln -sf ../../../lib/librecad/${plugins}.so ${DESTDIR}/usr/share/librecad/plugins/${plugins}.so
+	done
```

Comment:
```suggestion
	for plugin in $DESTDIR/usr/lib/librecad/*.so ; do
		ln -sf ../../../lib/librecad/$(basename $plugin) ${DESTDIR}/usr/share/librecad/plugins/$(basename $plugin)
	done
```

Something like this should work, what do you think?

Reply:
I agree, looks simpler.

Reply:
Then how to rebase to single commit?